ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -54,8 +54,7 @@ azure4) export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.jso
 gcp) export GOOGLE_CLOUD_KEYFILE_JSON=${CLUSTER_PROFILE_DIR}/gce.json;;
 kubevirt) export KUBEVIRT_KUBECONFIG=${HOME}/.kube/config;;
 vsphere) ;;
-openstack) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
-openstack-vexxhost) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
+openstack|openstack-vexxhost) export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml ;;
```

Comment:
Any reason not to capture all openstack variants with a wildcard?
```suggestion
openstack*) export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml ;;
```

Reply:
good idea! done